src/saving/SaveList.cs

```diff
@@ -56,6 +58,17 @@ public override void _Process(float delta)
         if (AutoRefreshOnFirstVisible && !refreshedAtLeastOnce && IsVisibleInTree())
         {
             Refresh();
+            wasVisible = true;
+            return;
+        }
+
+        if (!IsVisibleInTree())
```

Comment:
This logic needs to be optional by an exported property, see `AutoRefreshOnFirstVisible` as an example.
Also I feel like this logic should be combined with the first visible refresh check.